ol8/profiles/pci-dss.profile

```diff
@@ -110,7 +110,13 @@ selections:
     - display_login_attempts
     - gid_passwd_group_same
     - grub2_audit_argument
-    - smartcard_auth
+    - package_opensc_installed
+    - var_smartcard_drivers=cac
```

Comment:
One more note, in RHEL8, there is a new driver for legacy CAC version 1 cards, which might need to be enabled too if these cards are in use.

Reply:
@Jakuje, thanks a lot for your CAC1 note, I'll add this to OL smartcards support research tasks. Suggested set of changes in general aimed to sync existing RHEL smartcards rules with supported features on OL.